lib/InnerTransRel.mli

```diff
@@ -0,0 +1,45 @@
+(** Implements a transitive reflexive relation. *)
```

Comment:
I am puzzled, are the represented relation reflexive?

If I do not miss something, the last function `to_transitive` does not produce a reflexive relation.